content/en/blog/2022/graduation/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Tekton Graduation
+likTitle: Tekton Graduation
+date: 2022-10-26
+author: "Andrea Frittoli"
+description: >
+  From founding project to graduated, a brief history of Tekton
+---
+
+We're very happy to announce that Tekton as reached graduated status within the
+Continuous Delivery Foundation (CDF). The CDF Technical Oversight Committee
+(TOC) conducted public voting decide on the graduation status for Tekton, and
+the result was unanimously positive. The Tekton community is very proud of the
+results of the vote and will continue working to make Tekton better and safer
+for its users.
+
+In this blog post I will explore a bit of the history of Tekton, its "road to
+graduation" and what this milestone means for the project.
+
+## Early Days
+
+The Tekton project has its roots in [Knative][knative], where it is initially
+called "Knative Build" and later "Knative Pipeline". The project is spun off in
+August 2018, when it got it's current name and a new home on GitHub as
+["tektoncd/pipeline"][pipeline-repo]. Here's one of the project very first
+commits:
+
+```git
+commit 49d2316d71e8c315e0a8fd76008bc2920f56b3c3
+Author: Christie Wilson <bobcatfish@gmail.com>
+Date:   Fri Aug 31 17:15:35 2018 -0700
+
+Add pipeline strawman example
+
+@dlorenc @ImJasonH @tejal29 @aaron-prindle and I have been working on a
+strawman proposal for adding a Pipeline CRD and also for possibly
+envolving the Build CRD into a slightly more generic Task CRD.
+
+This PR demonstrates some paper prototype examples of what it could look
+like to define pipelines using the CRDs described in the README.
+```
+
+A few months later, in March 2019, [Tekton is donated][cdf-creation-blog] to the
+newly formed [Continuos Delivery Foundation (CDF)][cdf]. 
+
+## Growing the community
+
+Since then the project has thrived thanks to a rich community of contributors.
+One of earlier repositories to be created is the community one, where the
+project documented its governance, code of conduct and contributing guidelines.
+
+At the end of 2019 Tekton the number of repositories had grown to eleven. A
+regular monthly release cadence is established - Pipeline reaches 9 releases; in
+March 2020 the beta version of the Tekton Pipeline API was
+[released][pipeline-beta-release]. Release automation is implemented using
+Tekton itself.
+
+The "results" and "chains" project were started, and in August the Tekton Hub
```

Comment:
Is there a date we can add i.e. The results and chains projects were started in 2020 or something like that